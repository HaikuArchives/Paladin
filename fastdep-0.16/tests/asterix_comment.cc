/***************************************************************************
*
* HEADER: *
* AUTHOR: *
* *
****************************************************************************
*
* DESCRIPTION: *
* *
****************************************************************************/

#include "a.h"
