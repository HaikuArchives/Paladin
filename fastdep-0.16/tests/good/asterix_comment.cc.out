asterix_comment.o: asterix_comment.cc \
	a.h \
	c.h \
	d.h
